package.json

```diff
@@ -85,10 +86,6 @@
     "yarn": "1.22.5"
   },
   "resolutions": {
-    "micromatch": "^4.0.2",
-    "meow": "^7.1.1",
-    "node-fetch": "^2.6.1",
-    "systeminformation": "^5.3.1",
-    "xmlhttprequest-ssl": "^1.6.2"
+    "micromatch": "^4.0.4"
```

Comment:
All of these resolutions were added to fix security vulnerabilities. The only one we still need is `micromatch`, which can be fixed when we update `rollup-plugin-compat`'s dependencies.

Reply:
Thanks for taking care of this

Reply:
It should be fixed with: https://github.com/salesforce/es5-proxy-compat/commit/012991ede3f48fe7b651b07f0c41afe7012bfd2a.

As a side note, I don't see the point of using `resolutions` in this repo when it comes to NPM audit issues, except when it's a major security vulnerability (eg. [eslint-scope](https://eslint.org/blog/2018/07/postmortem-for-malicious-package-publishes)). The `resolutions` property is only applied to this repository. It makes sense if we were shipping an application with tight dependency control, but in our case where we are shipping libraries.

Reply:
In this very particular case, it actually does work because `es5-proxy-compat` is a dev dependency that we bundle into `@lwc/engine-core`. but I agree; `resolutions` kind of rubs me the wrong way, unless it's for a really critical security vulnerability, and if it actually affects consumers.

I'll publish a new `es5-proxy-compat` so we can get rid of `resolutions` entirely.

Reply:
Updated all the deps and got rid of `resolutions`!